### PR TITLE
Add global search in Poke

### PR DESCRIPTION
**Pull request by flvndvd**
## Description

This PR:
- Moves data sources loading to SWR hook
- Add global search on Poke's tables

### CODE REVIEW

File: front/poke/swr/data_sources.ts [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  const dataSourceViewsFetcher: Fetcher<PokeListDataSources> = fetcher;
+  const { data, error, mutate } = useSWRWithDefaults(
+    `/api/poke/workspaces/${owner.sId}/data_sources`,
+    dataSourceViewsFetcher,
+    { disabled }
+  );
+
+  return {
+    data: useMemo(() => (data ? data.data_sources : []), [data]),
+    isLoading: !error && !data,
```
Comment (tdraier) [inline front/poke/swr/data_sources.ts:21]: Add `&& !disabled`, or use isLoading from useSwr() ?

File: front/migrations/db/migration_86.sql [hunk 2]
```diff
@@ -1,44 +1,30 @@
...
-   SELECT perform_migration(:'backfilled'::boolean);
-\else
-    \echo '!! Migration was NOT applied !!'
-    \echo 'The backfill script: <insert_script> is required before applying this migation. If you already did it, run psql with --set=backfilled=1 argument.'
-\endif
+ALTER TABLE "data_source_views"
+ALTER COLUMN "editedAt"
+DROP DEFAULT;
 
-DROP FUNCTION perform_migration(boolean);
```
Comment (tdraier) [inline front/migrations/db/migration_86.sql:44]: what is this for ?

**Review by tdraier — COMMENTED** [review]
lgtm

**Review by tdraier — APPROVED** [review]

File: front/migrations/db/migration_86.sql (hunk 2 above)
Comment (flvndvd) [reply front/migrations/db/migration_86.sql:44]: See context here: https://dust4ai.slack.com/archives/C050SM8NSPK/p1727091558869199

**Pull request merged**